### ISSUE TITLE
Feature request: set Title to file name (less .mp4)

### ISSUE THREAD

**Issue by xmarc999** [griff/metaz#201]
Have ability to set Title to file name (less .mp4 extension) for multiple files.

**Comment by alipsky** [comment]
Bump

**Comment by griff** [comment]
What exactly do you mean by this? Because Title is already set to filename when loading the file.

**Comment by xmarc999** [comment]
Yes, but that may not be desired final name for the file. And this comes into play when ripping a TV series. 
In my case, I am using WinX DVD Ripper for Mac. I can set a base file name, say "The Big Bang Theory - s02." WinX will then rip the DVD, appending the title numbers to the base file name for each show ripped from the disk, i.e. The Big Bang Theory - s02_T1.mp4, The Big Bang Theory - s02_T2.mp4, The Big Bang Theory - s02_T3.mp4, and so on. It is also setting the metadata Title to the same value as the filename (minus the "mp4" of course).
I can then use A Better File Rename to put the filename in a format that my Plex Media Server will recognize. ABFR will both change any text to a given constant, as well as generate sequential numbers at the end, so I can rename the filenames to The Big Bang Theory - s02e01.mp4, The Big Bang Theory - s02e02.mp4, The Big Bang Theory - s02e03.mp4, etc. But ABFR doesn't work with metadata.
So now I open the files with MetaZ. I'm guessing that because WinX already set a metadata Title, MetaZ will not override that when it opens the files, even if the file name differs from the Title. So, in MetaZ, I just highlight and copy the main filename and paste that into the Title field for each episode. Then I do a Write All and Bob's your uncle.
Perhaps an option that says "reset Title to filename"?
By the way, it seems that there is almost always something about a ripped file that needs fixing. I have used MetaZ on nearly every one of my movies and TV Shows, comprising some 4 terabytes of video. You have made life better, no mean accomplishment.

**Comment by Duseylicious** [comment]
I would like this feature.  Sometimes I have files that all have metadata that is bad, but the filenames are good.  I'd love to just overwrite them all. 

Another way to accomplish this would be a way to quickly clear the meta titles from a batch, and the reload them in. (however I don't see a way to clear all titles at once, since it's "disabled for multiple.")
